FHEM/14_SD_WS07.pm

```diff
@@ -274,24 +314,45 @@ sub SD_WS07_Attr(@)
   <ul>
   	 <li>state (T: H:)</li>
      <li>temperature (&deg;C)</li>
-     <li>humidity: (the humidity 10-99)</li>
+     <li>humidity: (the humidity 1-100)</li>
      <li>battery: (low or ok)</li>
      <li>channel: (the channelnumberf)</li>
   </ul>
   <br>
   <b>Attributes</b>
   <ul>
     <li>correction-temp<br>
-       Damit kann die Temperatur korrigiert werden. Z.B. mit 10 wird eine um 10 Grad hoehere Temperatur angezeigt.
+       This can be used to correct the temperature. for example: at 10, the temperature is 10 degrees higher.
```

Comment:
This offset can be used to correct the temperature. For example: 10 means, that the temperature is 10 °C higher.

Ist negativ auch möglich?